src/pages/K8s.vue

```diff
@@ -435,6 +446,30 @@ export default {
     handleError(key, message) {
       this.handleNotification('error', key, message);
     },
+    confirm(message) {
+      return ipcRenderer.invoke(
+        'show-message-box',
+        {
+          message,
+          type:     'question',
+          title:    'Rancher Desktop - Kubernetes Settings',
+          buttons:  ['Ok', 'Cancel'],
```

Comment:
Also note that with the buttons in that order, pressing "OK" will return 0, and pressing "Cancel" or the ESC key will return 1. The code above is handling the opposite....